docs/package.json

```diff
@@ -5,8 +5,10 @@
   "license": "Apache-2.0",
   "homepage": "https://gestalt.pinterest.systems/",
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
+    "dev": "(next dev) & (yarn dev-metadata)",
+    "dev-metadata": "yarn watch 'yarn build-metadata' ../packages/gestalt/src",
```

Comment:
This won't pick up on changes to DatePicker, correct? Can that be added to `watch`'s args?